main.go

```diff
@@ -225,11 +225,21 @@ func runCLIApp(c *cli.Context) (err error) {
 	}
 
 	logger.Infof("Start gcsfuse/%s for app %q using mount point: %s\n", getVersion(), flags.AppName, mountPoint)
-	logger.Infof("GCSFuse mount command flags: %s", util.Stringify(*flags))
-	logger.Infof("GCSFuse mount config flags: %s", util.Stringify(*mountConfig))
+
+	flagsStringified, err := util.Stringify(*flags)
+	if err != nil {
+		logger.Warnf("failed to stringify cli flags: %v", err)
+	}
+	logger.Infof("GCSFuse mount command flags: %s", flagsStringified)
```

Comment:
We can move this block in the else part. Same for below one.

Reply:
done